src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator started its engine and began to work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator turned off its engine and stopped working.");
```

Comment:
The stop message should also be more informative and specific to the 'Excavator'. For example: "Excavator has stopped working."